utils/utils.js

```diff
@@ -72,4 +74,43 @@ export function addRepeater(element, title) {
   tag.append(plusIcon);
 
   element.append(tag);
+}
+
+export async function decorateTextfield(row, type = 'text') {
```

Comment:
I think the extraction here needs to be a little more configurable -- currently the function presume it's "info-field". We might need the function to be more flexible when it comes to options. If you are basing off of the newest stage, maybe the decorateSWCTextField(row, extraOptions) function in the event-materials-component.js can be of help?